### PR TITLE
Update links in the README.md

### PR DESCRIPTION
**Pull request by moneroexamples**
Related issue: https://github.com/ethereum/aleth/issues/5280

Considerable number of links and shields were not working as they were still pointing to `cpp-ethereum`. The PR aims at fixing that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,15 +87,15 @@ if it does not work for your use-case.
 ### Install dependencies (Windows)
 
 We provide prebuilt dependencies to build the project. Download them
-with the [scripts\install_deps.bat](scripts\install_deps.bat) script.
+with the [scripts\install_deps.bat](scripts/install_deps.bat) script.
```
Comment (chfast) [inline README.md]: Bring back the \

Comment (moneroexamples) [reply README.md]: Sure, but the link will lead to 404 page on github, as its doing now. Is it really intended?

Comment (chfast) [reply README.md]: Yes, you are right. Sorry.

Comment (moneroexamples) [reply README.md]: No problem. Glad I could help.

**Pull request merged**